public/src/components/Contact/Contact.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { AiFillPhone } from 'react-icons/ai';
+
+
+
+
+
+function Contact() {
+
+
+    return (
+        <div>
+             <div className="row">
+                <div className="container">
+                    <div className="row">
+
+                       <div className="col-sm-12">
+                           <h2 className="blockHeading">Contact Us</h2>
```

Comment:
All text on this should be pulled in via the content I believe there is still an example in the App.js

Reply:
We can do that but someone needs to show me how we are putting content in and where.

Reply:
Sure message me and we'll find time to meet